vllm/v1/engine/processor.py

```diff
@@ -278,10 +280,21 @@ def process_inputs(
             else:
                 sorted_mm_inputs = individual_mm_inputs
 
+        # Generate prompt kv block hashes
+        prompt_kv_block_hashes = hash_request_tokens(
```

Comment:
Should we skip this if `not self.cache_config.enable_prefix_caching`?

Reply:
Ah good point. We definitely should.